client/app/instructor/verifystudents/verifystudents.html

```diff
@@ -0,0 +1,34 @@
+<navbar></navbar>
+    <div class="container-fluid">
```

Comment:
container fluid is inconsistent with the rest of the site for the main content- should be `container`